test/e2e/lib/jest/globalSetup.ts

```diff
@@ -0,0 +1,47 @@
+import { mkdir } from 'fs/promises';
+import path from 'path';
+import config from 'config';
+import { chromium } from 'playwright';
+
+// Save cookies for defaultUser.
+export default async (): Promise< void > => {
+	if ( process.env.SAVE_AUTH_COOKIES !== 'true' ) {
+		return;
+	}
+
+	const browser = await chromium.launch();
+	const calypsoBaseURL = config.get( 'calypsoBaseURL' );
+	const cookieBasePath = path.join( process.cwd(), 'cookies' );
```

Comment:
Nit: I'd use something based on `__dirname` instead of `process.cwd()` to get the path. Using the cwd is a bit brittle if the user is running it from an IDE or using something like `yarn workspace run`

Reply:
Noted and will be revised in next commit.